css/lessc.sh

```diff
@@ -1 +1 @@
-../../enyo/tools/lessc.sh ./package.js
+../../enyo/tools/lessc.sh ./package.js --ri
```

Comment:
This is somewhat minor but I think we should pull this out of this PR. If we want to make this kind of global change, it should be in its own JIRA & PR with appropriate justification.